tests/src/ut_test_kvp.c

```diff
@@ -587,6 +593,303 @@ void test_ut_kvp_bool(void)
     UT_ASSERT( result == false );
 }
 
+void test_ut_kvp_bool_on_main_yaml(void)
+{
+     bool result;
+
+    /* Negative Tests */
+    result = ut_kvp_getBoolField( gpMainTestInstance, "6/value" );
+    UT_ASSERT( result == false );
+
+    /* Positive Tests */
+    result = ut_kvp_getBoolField( gpMainTestInstance, "1/value" );
+    UT_ASSERT( result == true );
+
+    result = ut_kvp_getBoolField( gpMainTestInstance, "2.value" );
+    UT_ASSERT( result == true );
+
+    result = ut_kvp_getBoolField( gpMainTestInstance, "3/value" );
+    UT_ASSERT( result == true );
+
+    result = ut_kvp_getBoolField( gpMainTestInstance, "4.value" );
+    UT_ASSERT( result == true );
+
+    result = ut_kvp_getBoolField( gpMainTestInstance, "5/value" );
+    UT_ASSERT( result == true );
+
+}
+
+void test_ut_kvp_fieldPresent_on_main_yaml(void)
+{
+    bool result;
```

Comment:
what's different about this compared to `test_ut_kvp_bool_on_main_yaml`? they look the same.?